vendor/TraceKit/tracekit.js

```diff
@@ -455,29 +455,26 @@ TraceKit.computeStackTrace = (function computeStackTraceWrapper() {
         if (xhr.status === 200) {
           var source = xhr.responseText;
 
-          // We trim the source down to the last 300 characters here as
-          // sourceMappingURL is usually at the end of the file.
-          source = source.substr(source.length - 300);
+          // We trim the source down to the last 300 characters as sourceMappingURL is always at the end of the file.
+          // Why 300? To be in line with: https://github.com/getsentry/sentry/blob/4af29e8f2350e20c28a6933354e4f42437b4ba42/src/sentry/lang/javascript/processor.py#L164-L175
```

Comment:
You spotted where I got 300 from, nice 👍

Reply:
Haha, I'm surprised you found it! :D

Reply:
yeah, nice work @CaptObvious